xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-test/xwiki-platform-mentions-test-pageobjects/src/main/java/org/xwiki/mentions/test/po/MentionNotificationPage.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.mentions.test.po;
+
+import java.util.List;
+
+import org.openqa.selenium.WebElement;
+import org.xwiki.platform.notifications.test.po.GroupedNotificationElementPage;
+
+import static org.openqa.selenium.By.cssSelector;
+import static org.openqa.selenium.By.xpath;
+
+/**
+ * Mentions notifications page object.
+ *
+ * @version $Id$
+ * @since 12.7
+ * @since 12.6.2
```

Comment:
Here too.

Reply:
Basically the only since you should put here for now is 12.8RC1: we're not sure yet to backport this to branches.